modules/rostests/winetests/kernel32/path.c

```diff
@@ -1935,11 +1935,15 @@ static void test_SearchPathW(void)
     static const WCHAR extW[] = {'.','e','x','t',0};
     static const WCHAR dllW[] = {'.','d','l','l',0};
     static const WCHAR fileW[] = { 0 };
+    static const WCHAR* longDirName =  L"C:\\Program Files\\Very_long_test_path_which_can_trigger_heap_overflow_test_1234567890______________________________________________________AB";
+    static const WCHAR* longFileName = L"this_is_long_file_name_for_checking______________________________________________________________________________CD";
+    static const WCHAR* ext = L".txt";
+
```

Comment:
This is a winetest, which is synced with Wine. Better to add new tests to rostests (or use ifdef guard).

Reply:
Ok.